### PR TITLE
update(pol): polish faqs

### PR DESCRIPTION
**Pull request by DannyS03**
## Context

Refines FAQs to address nuanced considerations and respond to general feedback

### CODE REVIEW

File: docs/pos/assets/pol.md [hunk 1]
```diff
@@ -21,15 +21,18 @@ There is a proposal to transition the native token of the Polygon PoS network fr
 
 ### What is POL?
 
-POL is an upgrade of the native token for the Polygon PoS network, designed to replace the existing MATIC token. It serves multiple purposes, including governance, staking, and transaction fees within the Polygon ecosystem. Importantly, POL has the same rights and features as MATIC, is built on OpenZeppelin's ERC20 implementations, and supports EIP-2612 for signature-based permit approvals.
+POL is native token upgrade for the Polygon ecosystem, designed to support a wide range of activities and purposes,
```
Comment (cerberushades) [inline docs/pos/assets/pol.md:24]:
```suggestion
POL is the native token upgrade for the Polygon ecosystem, designed to support a wide range of activities and purposes, 
```

File: docs/pos/assets/pol.md [hunk 2]
```diff
@@ -21,15 +21,18 @@ There is a proposal to transition the native token of the Polygon PoS network fr
 
 ### What is POL?
 
-POL is an upgrade of the native token for the Polygon PoS network, designed to replace the existing MATIC token. It serves multiple purposes, including governance, staking, and transaction fees within the Polygon ecosystem. Importantly, POL has the same rights and features as MATIC, is built on OpenZeppelin's ERC20 implementations, and supports EIP-2612 for signature-based permit approvals.
+POL is native token upgrade for the Polygon ecosystem, designed to support a wide range of activities and purposes, 
+including being used as a tool for network participation and security. By staking POL, participants can actively contribute 
+to the ecosystem as validators. Importantly, POL has many of the and features as MATIC, is built on OpenZeppelin's ERC20
```
Comment (cerberushades) [inline docs/pos/assets/pol.md]:
```suggestion
to the ecosystem as validators. Importantly, POL has as many features as MATIC; it is built on OpenZeppelin's ERC20 
```

File: docs/pos/assets/pol.md [hunk 3]
```diff
@@ -89,17 +91,20 @@ Yes, the migration contract includes a feature known as "**unmigration**." This
 
 ## Bridging Mechanisms
 
-### Is There a Possibility of Funds Getting Locked or Stuck?
+### How Does the Modified Plasma Bridge Function?
 
-Yes, if an existing contract relies on withdrawing MATIC from a bridge and receives POL instead, this might result in locked funds. **Developers must check their contracts, verify the transaction lifecycle, and engage on the forum for any doubts.**
+The Plasma Bridge will undergo modifications, with community approval, to change the native of Polygon PoS to the new POL token. Specifically, the following changes are being proposed:
```
Comment (cerberushades) [inline docs/pos/assets/pol.md:96]:
```suggestion
The Plasma Bridge will undergo modifications, with community approval, to change the native token of Polygon PoS to the new POL token. Specifically, the following changes are being proposed:
```

**Review by cerberushades — CHANGES_REQUESTED** [review]

File: docs/pos/assets/pol.md (hunk 2 above)
Comment (DannyS03) [reply docs/pos/assets/pol.md]: Updated in https://github.com/0xPolygon/wiki/pull/142/commits/9f0753f2ba481cfb2a2845e1d1dfd388ef092031

**Review by cerberushades — APPROVED** [review]

**Pull request merged**